src/components/DatePicker/index.android.js

```diff
@@ -1,94 +1,86 @@
-import React from 'react';
+import React, {useState, forwardRef} from 'react';
 import {Keyboard} from 'react-native';
 import RNDatePicker from '@react-native-community/datetimepicker';
 import moment from 'moment';
-import _ from 'underscore';
+import isFunction from 'lodash/isFunction';
```

Comment:
Any reason for replacing underscore with lodash?

Reply:
It was suggested by a review from our team, but I just reverted it back to underscore

Reply:
<img width="826" alt="Screenshot 2023-09-21 at 10 51 59" src="https://github.com/Expensify/App/assets/57314631/52c01ee6-4a4f-461a-9f73-6e404fc95020">

Reply:
Gotcha. But we haven't ever used `lodash/isFunction` pattern in our app. Just don't wanna be the first introducer.